src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +208,555 @@
+
+                <controls:SettingsGroup x:Uid="General_SettingsBackupAndRestore" Visibility="Visible">
+                    <controls:SettingExpander IsExpanded="True">
+                        <controls:SettingExpander.Header>
+                            <controls:Setting x:Uid="General_SettingsBackupAndRestore" Icon="&#xE777;" Style="{StaticResource ExpanderHeaderSettingStyle}">
+                                <controls:Setting.Description>
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_BackupAndRestoreDescription" />
+                                    </TextBlock>
                                 </controls:Setting.Description>
                                 <controls:Setting.ActionContent>
-                                    <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.ThemeIndex}" MinWidth="{StaticResource SettingActionControlMinWidth}">
-                                        <ComboBoxItem x:Uid="Radio_Theme_Dark"/>
-                                        <ComboBoxItem x:Uid="Radio_Theme_Light"/>
-                                        <ComboBoxItem x:Uid="Radio_Theme_Default"/>
-                                    </ComboBox>
+
+
+                                    <Grid Visibility="Visible">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="1*" />
+                                            <ColumnDefinition Width="1*" />
+                                        </Grid.ColumnDefinitions>
+                                        <Button Grid.Column="0" x:Uid="GeneralPage_BackupConfigs"
+                                        Command="{Binding BackupConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                                        <Button Grid.Column="2" x:Uid="GeneralPage_RestoreConfigs"
+                                        Command="{Binding RestoreConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                                    </Grid>
                                 </controls:Setting.ActionContent>
                             </controls:Setting>
-                    <controls:Setting x:Uid="GeneralPage_RunAtStartUp">
+                        </controls:SettingExpander.Header>
+
+                        <controls:SettingExpander.Content>
+
+
+                            <StackPanel Orientation="Vertical">
+
+                                <controls:Setting x:Uid="General_SetupBackupAndRestoreLocationText" Visibility="Visible">
```

Comment:
Style="{StaticResource ExpanderContentSettingStyle}" is missing. This makes sure we have consistency across all SettingsExpanders and its items.

Reply:
WPF/XAML takes me a while to get "right" and there are many ways to do the same thing. I think "could be better" can wait, and perhaps done by those like you which are clearly better at it than me.

Reply:
OK, will add that now.